test/sqlancer/dbms/TestStoneDB.java

```diff
@@ -0,0 +1,22 @@
+package sqlancer.dbms;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
+
+import org.junit.jupiter.api.Test;
+
+import sqlancer.Main;
+
+public class TestStoneDB {
+
+    String stoneDBAvailable = System.getenv("STONEDB_AVAILABLE");
+    boolean stoneDBIsAvailable = stoneDBAvailable != null && stoneDBAvailable.equalsIgnoreCase("true");
+
+    @Test
+    public void testStoneDB() {
+        assumeTrue(stoneDBIsAvailable);
+        assertEquals(0, Main.executeMain("--random-seed", "0", "--timeout-seconds", TestConfig.SECONDS, "--num-threads",
+                "1", "--num-queries", TestConfig.NUM_QUERIES, "stonedb", "--oracle", "NoRec"));
```

Comment:
Did you successfully run it? I guess it should be`NoREC` instead of `NoRec`?

Reply:
Yes, make sense. will change.